containers/Dockerfile.EESSI-build-node-debian10

```diff
@@ -1,5 +1,6 @@
 ARG cvmfsversion=2.8.2
```

Comment:
@bedroge Shall we bump the CernVM-FS version to the latest (2.9.1) too while we're at it?

Reply:
Good point, done!

Reply:
Should be 2.9.0 though ;-)

Reply:
Oh, I could've sworn that CernVM-FS 2.9.1 was released already... 🤔 

That means we won't have the GeoAPI Python 3 fix (https://github.com/cvmfs/cvmfs/pull/2829) yet, but that's really only relevant for Stratum-1's?

Reply:
Yep, that should be fixed on the Stratum 1. ETA for 2.9.1 is beginning of Feb.